README.adoc

```diff
@@ -1 +1,20 @@
-# tensorflow
+# Spring Cloud Stream TensorFlow Processor
+
+A processor that evaluates a machine learning models build with link:https://www.tensorflow.org/[TensorFlow] and stored
```

Comment:
thanks @artembilan! i've started applying the suggested changes. At least the obvious one. 
is it ok if i apply them in multiple commits or i should wait until all questions and request are cleared and then make a single commit with all changes?